pkgs/tools/networking/snowflake/default.nix

```diff
@@ -1,20 +1,26 @@
-{ lib, buildGoModule, fetchgit }:
+{ lib, buildGoModule, fetchFromGitLab }:
 
 buildGoModule rec {
   pname = "snowflake";
-  version = "2.3.1";
+  version = "2.4.1";
 
-  src = fetchgit {
-    url = "https://git.torproject.org/pluggable-transports/${pname}";
+  src = fetchFromGitLab {
+    domain = "gitlab.torproject.org";
+    group = "tpo";
+    owner = "anti-censorship/pluggable-transports";
+    repo = "snowflake";
     rev = "v${version}";
-    sha256 = "sha256-4/ZTLyST73krOL87am28TM+1mktchpoCSaASMqQl5e8=";
+    sha256 = "sha256-DR1H5ncFPHZWQAwOZKkfRrjwfzhirSzwtvKesaRmqcA=";
   };
 
-  vendorSha256 = "sha256-a2Ng+D1I0v5odChM6XVVnNwea/0SOTOmdm2dqKaSU3s=";
+  vendorHash = "sha256-66GqvwHPkMii5oXZV36ayYYkW1oaq5qTjkEA5BeS/5U=";
+
+  ldflags = [ "-s" "-w" ];
```

Comment:
```suggestion
```
That's fair but it's not agreed upon across nixpkgs so we have to keep consistency.

Reply:
Sure, fixed!